### DISCUSSION TITLE
两个问题

### DISCUSSION THREAD

**Discussion by OrdChaos** [Clipteam/clipcc-gui#145] [你问我答/Q&A]
第一，社区如何转正？我没看到问卷欸......
第二，别针社区设计页设置了60帧运行并正常，但作品页仍然只有30帧，是bug吗？

**Comment by elvisdragonmao** [comment]
是否方便提供作品連結?

**Comment by SimonShiki** [comment]
> 第一，社区如何转正？我没看到问卷欸......
> 第二，别针社区设计页设置了60帧运行并正常，但作品页仍然只有30帧，是bug吗？

第一，在群里，后续考虑移到社区内
第二，作品帧率由作品本身而不是个人决定
